### ISSUE TITLE
Can't resolve `./null-prototype` when importing `@webcontainer/api@1.4.0` into a Webpack 5 app

### ISSUE THREAD

**Issue by fengyuanchen** [stackblitz/webcontainer-core#1547]
### Describe the bug

 When importing `@webcontainer/api@1.4.0` into a project using Webpack 5 bundler, it throws an error:

```shell
ERROR in ./node_modules/@webcontainer/api/dist/utils/file-system.js 1:0-49
Module not found: Error: Can't resolve './null-prototype' in '/root/myagent/_work/7671/s/node_modules/@webcontainer/api/dist/utils'
Did you mean 'null-prototype.js'?
BREAKING CHANGE: The request './null-prototype' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
resolve './null-prototype' in '/root/myagent/_work/7671/s/node_modules/@webcontainer/api/dist/utils'
  using description file: /root/myagent/_work/7671/s/node_modules/@webcontainer/api/package.json (relative path: ./dist/utils)
    Field 'browser' doesn't contain a valid alias configuration
    using description file: /root/myagent/_work/7671/s/node_modules/@webcontainer/api/package.json (relative path: ./dist/utils/null-prototype)
      Field 'browser' doesn't contain a valid alias configuration
      /root/myagent/_work/7671/s/node_modules/@webcontainer/api/dist/utils/null-prototype doesn't exist
 @ ./node_modules/@webcontainer/api/dist/utils.js 5:0-92 5:0-92 5:0-92
 @ ./node_modules/@webcontainer/api/dist/index.js 14:0-98 17:0-27 17:0-27 109:19-43 117:20-36 137:48-72
 @ ./src/utils/web-container.ts
 @ ./src/utils/index.ts 8:0-32 8:0-32
 @ ./src/main.ts 10:0-30 19:4-7

webpack 5.92.1 compiled with 1 error and 3 warnings in 347007 ms
```

### Link to the blitz that caused the error

null

### Steps to reproduce

1. Open the `@webcontainer/api/dist/utils/file-system.js` file.
2. The first line is `import { nullPrototype } from './null-prototype';` (the `.js` extension is missing).
3. Import `@webcontainer/api` in Webpack 5 app, it throws errors in the shell.

### Expected behavior

1. Open the `@webcontainer/api/dist/utils/file-system.js` file.
2. The first line is `import { nullPrototype } from './null-prototype.js';` (the `.js` extension is existing).
3. Import `@webcontainer/api` in Webpack 5 app, it works fine.

### Parity with Local

- [X] I have [run the project in my local machine](https://github.com/stackblitz/webcontainer-core/blob/main/repro.md#try-locally) and I could not reproduce the issue.

### Platform

- OS: Windows 10
- Browser: Chrome
- Version: 122

**Comment by khanakia** [comment]
I am facing the same issue with remix app
```
1:16:18 PM [vite] Internal server error: Cannot find module '/blitz_webcontainers_remix/node_modules/@webcontainer/api/dist/utils/null-prototype' imported from /blitz_webcontainers_remix/node_modules/@webcontainer/api/dist/utils/file-system.js
      at finalizeResolution (node:internal/modules/esm/resolve:257:11)
      at moduleResolve (node:internal/modules/esm/resolve:913:10)
      at defaultResolve (node:internal/modules/esm/resolve:1037:11)
      at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:650:12)
      at #cachedDefaultResolve (node:internal/modules/esm/loader:599:25)
      at ModuleLoader.resolve (node:internal/modules/esm/loader:582:38)
      at ModuleLoader.getModuleJobForImport (node:internal/modules/esm/loader:241:38)
      at ModuleJob._link (node:internal/modules/esm/module_job:132:49) (x14)
```

**Comment by alexander-albul** [comment]
Same issue with pnpm and svelte 5. Adding '.js' helped, thanks

**Comment by khanakia** [comment]
@alexander-albul how did you resolve the issue? 

Will you elaborate more, please? Where did you the `.js` file

**Comment by fengyuanchen** [comment]
Just found out that this issue has been fixed in v1.5.1.
